pkg/config/config_template.yaml

```diff
@@ -782,7 +782,7 @@ api_key:
 
     ## @param privProtocol - string - optional
     ## The privacy protocol to use when connecting to your SNMP devices.
-    ## Available options are: DES, 3DES, AES, AES192, AES256, AES192C, AES256C.
+    ## Available options are: DES, AES, AES192, AES192C, AES256, AES256C
```

Comment:
Note for reviewers: 3DES is only available for Python, so removing it.

3DES seems to be a workaround for:

> 		// Many vendors, including Cisco, use the 3DES key extension algorithm to extend the privacy keys that are too short when using AES,AES192 and AES256.

gosnmp seems to handle keys that are too short here:

https://github.com/gosnmp/gosnmp/pull/223/files#diff-5339dca39e1d226e550eba44e438e31157f1b05e5d937fc3133f6029051e2501R327-R331